checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/api/session/SessionRequest.kt

```diff
@@ -4,10 +4,16 @@ import java.io.Serializable
 
 interface SessionRequest: Serializable
 
+/**
+ * [CardSessionRequest] is a serializable class that contains the request data for a VT session
+ */
 internal data class CardSessionRequest(val cardNumber: String, val cardExpiryDate: CardExpiryDate, val cvv: String, val identity: String): SessionRequest {
 
     internal data class CardExpiryDate(val month: Int, val year: Int): Serializable
 
 }
 
+/**
+ * [CVVSessionRequest] is a serializable class that contains the request data for a Payments CVC session
```

Comment:
Worth mentioning that this **represents** the request body for the Payments CVC session

Reply:
updated